app/views/splash/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render partial: 'shared/nav', locals: { h1_content: "Budget-App" } %>
+<div class='splash'>
+    <h1>Welcome to Budget-App</h1>
+    <p>Get started by signing up or logging in:</p>
+    <div id='splash_link'>
+        <a href="<%= new_user_registration_path %>">Sign Up</a>
+        <a href="<%= new_user_session_path %>">Log In</a>
+    </div>
```

Comment:
- [ ] Kindly style the sign-in/sign-up links to match the given design, I have attached an image for your reference.


![splash](https://github.com/oguntayomicheal/Budget-App/assets/53124548/488842b7-1558-4d55-9603-ac59c617e1fc)


![signup](https://github.com/oguntayomicheal/Budget-App/assets/53124548/63ad5554-6698-413c-be67-85538cb2e37f)